libs/components/modals/src/lib/modules/modal/modal.component.scss

```diff
@@ -106,15 +106,19 @@
 
 .sky-modal-header {
   flex-shrink: 0;
+  position: relative;
   z-index: 2;
 }
 
 .sky-modal-content {
   overflow-y: auto;
+  position: relative;
```

Comment:
Can you give more context around the `position: relative` rules? This might cause adverse effects to our consumers if they're using `position: absolute` etc. for elements in their modals.